extensions/browser-history/src/components/actions.tsx

```diff
@@ -0,0 +1,55 @@
+import { Action, ActionPanel, Detail, Icon, openCommandPreferences } from "@raycast/api";
+import { openNewChromeTab, openNewFirefoxTab, openNewSafariTab } from "../actions";
+import { HistoryEntry } from "../interfaces";
+
+export const HistoryItemAction = ({ entry: { title, url } }: { entry: HistoryEntry }) => (
+  <ActionPanel title={title}>
```

Comment:
I would remove the title from here since it can be very long

Reply:
It has been removed.